pokedex/pokemon.lua

```diff
@@ -1073,7 +1081,8 @@ function M.upgrade_pokemon(pkmn)
 				if not pkmn.variant then
 					pkmn.variant = pokedex.get_default_variant(M.get_current_species(pkmn))
 				end
-				
+			elseif i == 2 then
```

Comment:
This may need to be updated to `i == 3` after merging from master, be on the lookout for that.